cdk/lib/__snapshots__/gatehouse.test.ts.snap

```diff
@@ -907,14 +925,62 @@ exports[`The Gatehouse stack matches the snapshot 1`] = `
               "Fn::Join": [
                 "",
                 [
-                  "#!/bin/bash
-mkdir -p $(dirname '/gatehouse/gatehouse.deb')
-aws s3 cp 's3://",
+                  "#!/bin/bash -ev
+# Install X-Ray Collector
+wget -P /tmp https://aws-otel-collector.s3.amazonaws.com/ubuntu/arm64/latest/aws-otel-collector.deb
+dpkg -i /tmp/aws-otel-collector.deb
+cat << EOF > /opt/aws/aws-otel-collector/etc/config.yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+processors:
+  resourcedetection/ec2:
+    detectors:
+      - ec2
+    ec2:
+      tags:
+        - Stage
+    timeout: 2s
+    override: false
+  memory_limiter:
+    check_interval: 1s
+    limit_mib: 20
+  batch/traces:
+    timeout: 1s
+    send_batch_size: 50
+exporters:
+  awsxray:
+    indexed_attributes:
+      - otel.resource.ec2.tag.Stage
+extensions:
+  awsproxy:
+service:
+  extensions:
+    - awsproxy
+  pipelines:
+    traces:
+      receivers:
+        - otlp
+      processors:
+        - resourcedetection/ec2
+        - memory_limiter
+        - batch/traces
+      exporters:
+        - awsxray
+EOF
```

Comment:
another option would be to put this config.yaml as a file in the repo that gets included in the deployed package